servicex/query.py

```diff
@@ -456,7 +491,8 @@ async def get_signed_url(
             # are guaranteed to be in the bucket. Also, if we are just downloading or
             # signing urls for a previous transform then we know it is complete as well
             if cached_record or (
-                self.current_status and self.current_status.status == Status.complete
+                self.current_status
+                and self.current_status.status in (Status.complete, Status.canceled, Status.fatal)
```

Comment:
I wonder if we shouldn't have a constant that is set `sx_status_done = (Status.complete, Status.canceled, Satus.fatal)` and then we can do things like `self.current_status.status in sx_status_done` or similar - since this gets repeated several times.

Reply:
Probably a good idea - if we introduce new status codes (e.g. "lookup failed") this could get unwieldy writing it out.